digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectValidatorSpec.scala

```diff
@@ -1640,16 +1640,90 @@ class ProjectValidatorSpec extends FunSuite with Matchers {
       validationErrors3.foreach(e => e.validationError.value should be(projectValidator.ValidationErrorList.ErrorInValidationOfUnchangedLinks.value))
     }
   }
-   test("Test checkForInvalidUnchangedLinks When middle of road part is changed to another road number then there cant be unchanged after the transfer") {
+
+  test("Test checkForInvalidUnchangedLinks When project link with status New is added to the start of the road part then there cant be links with status Unchanged after that") {
```

Comment:
"---then there cant be links with status Unchanged after that"
 joko
-nillinnii, typo:  "can't" :D
tai negaation kierto lauseen ymmärrettävyyden parantamiseksi:
-"--- then links with status Unchanged are forbidden after that"
tai ehkä jopa lisäksi myös tuon that:in eksplikointi:
-"--- then links with status Unchanged are forbidden after the new link"